portfolio/src/main/webapp/education.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Ishani Santurkar's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="content">
+    <header>
+        <a class="headerelem" href="index.html">About Me</a>
```

Comment:
nit: use 2-space indentation 
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Unfortunately cloud shell does not auto-format HTML correctly, so you either need to manually format (which is boring) or set up a local IDE (which might be useful for later). VSCode is probably a good choice if you choose the later. Let me know if you need help for this!

Reply:
I don't think you can install VScode on cloud shell -- the way you did it is exactly what I mean.